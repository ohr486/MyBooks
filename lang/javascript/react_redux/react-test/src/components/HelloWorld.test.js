import React from 'react';
import renderer from 'react-test-renderer';
import HelloWorld from './HelloWorld';

test('Hello Worldコンポーネントのスナップショットテスト', () => {
    const result = renderer.create(<HelloWorld />).toJSON();
    expect(result).toMatchSnapshot();
});