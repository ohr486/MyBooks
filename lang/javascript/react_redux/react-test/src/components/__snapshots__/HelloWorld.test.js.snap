// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello Worldコンポーネントのスナップショットテスト 1`] = `null`;
